components/core/src/clp/DictionaryReader.hpp

```diff
@@ -131,17 +120,7 @@ void DictionaryReader<DictionaryIdType, EntryType>::open(
         throw OperationFailed(ErrorCode_NotReady, __FILENAME__, __LINE__);
     }
 
-    constexpr size_t cDecompressorFileReadBufferCapacity = 64 * 1024;  // 64 KB
-
-    open_dictionary_for_reading(
-            dictionary_path,
-            segment_index_path,
-            cDecompressorFileReadBufferCapacity,
-            m_dictionary_file_reader,
-            m_dictionary_decompressor,
-            m_segment_index_file_reader,
-            m_segment_index_decompressor
-    );
+    read_new_entries(dictionary_path, segment_index_path);
```

Comment:
Make it to be a separate function so the code looks cleaner. we can also put code directly into open and remove the function?